src/manifest.model.yaml

```diff
@@ -11,10 +11,14 @@ applications:
 models:
 - UNet2DNucleiBroad:
   config_url: https://raw.githubusercontent.com/bioimage-io/pytorch-bioimage-io/master/specs/models/unet2d/nuclei_broad/UNet2DNucleiBroad.model.yaml
-  applications: [Fiji, Ilastik, ImJoy]
-  download_url: https://github.com/bioimage-io/pytorch-bioimage-io/archive/master.zip
+  applications: [Ilastik, ImJoy]
+  download_url: https://github.com/bioimage-io/pytorch-bioimage-io/model_packages/UNet2DNucleiBroad.model.zip
```

Comment:
Can you do a release on the repo and upload zip file to the release page? I think that's a better way to host zip files.